types/apimocker/package.json

```diff
@@ -0,0 +1,3 @@
+{
+    "private": true
```

Comment:
You don’t need to add a package.json unless you have external dependencies to put in here—you can simply delete the file.